### PR TITLE
feat: Create schema for ingredient entry 

### LINKED ISSUE #102: FeedAHelp: Create Schema in Sanity for Ingredient Entry
Depends on #101 

```
export default defineType({
  name: 'ingredients',
  title: 'Ingredients',
  type: 'document',
  icon: MenuIcon,
  fields: [
    defineField({
      name: 'name',
      type: 'string',
      title: 'Ingredient Name',
      validation: (Rule) => Rule.required(),
    }),
   .
   .
   .
  ],
})
```

### PR DESCRIPTION
**Pull request by biplob2358**
# Description

Schema for ingredient entry 

Fixes #102 

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

Closes #102 

image

### CODE REVIEW

File: apps/feedahelp-cms/schemas/index.ts [hunk 1]
```diff
@@ -1,5 +1,4 @@
 import fahIngredients from './fahIngredients'
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/index.ts]: what is `fahIngredients`? Fahmid Ingredients? Fashion Ingredients?

File: apps/feedahelp-cms/schemas/index.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
...
+import ingredient from './ingredient'
 
-export const schemaTypes = [
-    fahIngredients
-]
+export const schemaTypes = [fahIngredients, ingredient]
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/index.ts]: what is fahIngredients? Fahmid Ingredients? Fashion Ingredients?

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import {LemonIcon} from '@sanity/icons'
+import {defineField, defineType} from 'sanity'
+
+export default defineType({
+  name: 'ingredient',
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: `ingredients` => plural

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+  title: 'Ingredient',
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: ingredients => plural

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+  icon: LemonIcon,
+  fields: [
+    defineField({
+      name: 'ingredientName',
+      title: 'Ingredient Name',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: 'ingredientNameBangla',
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: rename this `banglaName`

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+      title: 'Ingredient Name Bangla',
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: just `Bangla Name`

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 7]
```diff
@@ -0,0 +1,47 @@
...
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name: 'ingredientImage',
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: just `image`

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 8]
```diff
@@ -0,0 +1,47 @@
...
+        title: 'Ingredient Image',
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: just `Image`, you see we are already in Ingredients, you don't have name everything ingredients

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 9]
```diff
@@ -0,0 +1,47 @@
...
+        title:'Price',
+        type:'number',
+        validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name: 'status',
+        title: 'Status',
+        type: 'boolean',
+        validation: (Rule) => Rule.required(),
+    }),
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: We need a new `measuringQty` =>

example:
The units needed for ingredient calculation qty can vary depending on the ingredient and the recipe. However, some common units used include:

- Grams (g)
- Milliliters (mL)
- Ounces (oz)
- Cups (c)
- Tablespoons (Tbsp)
- Teaspoons (tsp)
- Dash
- Pinch
- Piece
- Clove
- Sprig
- Stem

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
do the changes and come back

File: apps/feedahelp-cms/schemas/ingredient.ts [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+      title: 'Ingredient Name Bangla',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+        name: 'ingredientImage',
+        title: 'Ingredient Image',
+        type: 'image',
+        validation: (Rule) => Rule.required(),
+    }),
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredient.ts]: We need another `imgURL`
https://drive.google.com/file/d/1PgJJFhdHbzBtWIzUxO3l2Yk94x3arKGs/view?usp=drive_link

**Review by TanvirAlam — CHANGES_REQUESTED** [review]

File: apps/feedahelp-cms/schemas/index.ts (hunk 2 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/index.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredient.ts (hunk 4 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/ingredient.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredient.ts (hunk 5 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/ingredient.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredient.ts (hunk 6 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/ingredient.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredient.ts (hunk 7 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/ingredient.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredient.ts (hunk 8 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/ingredient.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredient.ts (hunk 9 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/ingredient.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredient.ts (hunk 10 above)
Comment (biplob2358) [reply apps/feedahelp-cms/schemas/ingredient.ts]: resolved

File: apps/feedahelp-cms/schemas/ingredients.ts [hunk 11]
```diff
@@ -0,0 +1,57 @@
...
+      title: 'Ingredient Name',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
+    defineField({
+      name: 'banglaName',
+      title: 'Bangla Name',
+      type: 'string',
+      validation: (Rule) => Rule.required(),
+    }),
```
Comment (TanvirAlam) [inline apps/feedahelp-cms/schemas/ingredients.ts:21]: where is the English name?

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
one more

**Review by TanvirAlam — APPROVED** [review]
LGTM

**Pull request merged**